src/consensus/pbft_manager.cpp

```diff
@@ -1524,9 +1558,8 @@ bool PbftManager::giveUpNextVotedBlock_() {
     }
     // Read from DB pushing into unverified queue
     pbft_chain_->pushUnverifiedPbftBlock(pbft_block);
-  }
-
-  if (pbft_block) {
+  } else {
```

Comment:
I think cannot change here. The previous approach could handle the PBFT block doesn't in the unverified queue but in the DB

Reply:
You are right, fixed it.